pyproject.toml

```diff
@@ -192,20 +192,16 @@ exclude = [
 [tool.ruff.pydocstyle]
 convention = 'numpy'
 
-[tool.pytest]
-python_files = [
-    'benchmark_*.py',
-    'test_*.py',
-]
-python_classes = [
-    'Test*',
-    '*Suite',
-]
-python_functions = [
-    'time_*',
-    'test_*',
-    'peakmem_*',
-]
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "-ra  --strict-config  --strict-markers"
+log_cli_level = "INFO"
+xfail_strict = true
+#testpaths = ["skimage", "../../../../../benchmarks"]
+python_files = ["benchmark_*.py", "test_*.py"]
+python_classes = ["Test*", "*Suite"]
+python_functions = ["time_*", "test_*", "peakmem_*"]
+filterwarnings = []
```

Comment:
Does anyone mind if I move our custom implementation of warning filters to this configuration? 

https://github.com/scikit-image/scikit-image/blob/89fa48aabdd96d7b6ae47dd111393564b7488384/skimage/_shared/testing.py#L209-L299